components/b2b/seller/AccountsTable/AccountsTable.tsx

```diff
@@ -78,20 +84,19 @@ const AccountsTable = (props: AccountsTableProps) => {
   const [searchTerm, setSearchTerm] = useState('')
   const debouncedTerm = useDebounce(searchTerm, publicRuntimeConfig.debounceTimeout)
 
-  // TODO: Implement filter
-  // const handleFilterAction = (filters: QuoteFilters) => {
-  //   setQuotesSearchParam({ filter: buildQuotesFilterParam(filters) })
-  // }
-
-  // const handleFilterButtonClick = () => {
-  //   showModal({
-  //     Component: QuotesFilterDialog,
-  //     props: {
-  //       filters: filters,
-  //       // onFilterAction: handleFilterAction,
-  //     },
-  //   })
-  // }
+  const handleFilterAction = (filters: B2bContactsFilters) => {
```

Comment:
Please write the test cases for filter and search functionality added.

Reply:
Added